### PR TITLE
Updated version number

### PR DESCRIPTION
**Pull request by deepsidhu85**
Updated the version number in:

1. CHANGELOG
2. setup.py
3. irida_import.xml.sample

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 All notable changes to irida-galaxy-importer will be documeted in this file.
 
+## 2.1.0
+* Added in support for importing IRIDA files that may be stored in the cloud.
```
Comment (ericenns) [inline CHANGELOG.md]: Could this be updated to state that it adds supports for IRIDA files that are not available locally (i.e. in the cloud)

**Review by ericenns — CHANGES_REQUESTED** [review]
Just a minor comment about the changelog otherwise it looks good.

File: CHANGELOG.md (hunk 1 above)
Comment (deepsidhu85) [reply CHANGELOG.md]: Sure! Updated in 62a41768af58e85053d84f4228e1e56642571e4a

**Review by ericenns — APPROVED** [review]
Looks good to me!

**Pull request merged**